src/exchange/store.go

```diff
@@ -153,7 +159,7 @@ func (s *Store) BindAddress(skyAddr, depositAddr, coinType string) error {
 			}
 		}
 
-		addrs = append(addrs, depositAddr)
+		addrs = append(addrs, DepositElement{DepositAddr: depositAddr, CoinType: coinType})
```

Comment:
This breaks format of existing db, any other way?

Reply:
i can find address type by search used_xxx_address buckets, that is ok?

Reply:
or use a new bucket to store relationship between address and type

Reply:
Store bind addresses in a different bucket per coin, seems best